src/Test.UnitTests.Sarif/TestData/InsertOptionalDataVisitor/ExpectedOutputs/CoreTests.sarif

```diff
@@ -1,141 +1,118 @@
 {
-  "$schema": "http://json.schemastore.org/sarif-2.0.0",
-  "version": "2.0.0",
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.1.json",
+  "version": "2.1.0",
   "runs": [
     {
       "tool": {
-        "name": "Sarif.UnitTests",
-        "fullName": "Sarif.UnitTests.1.3.3.0-beta",
-        "version": "1.3.3.0",
-        "semanticVersion": "1.3.3",
-        "sarifLoggerVersion": "1.5.22.0"
+        "driver": {
+          "name": "Sarif.UnitTests",
+          "fullName": "Sarif.UnitTests.1.3.3.0-beta",
+          "version": "1.3.3.0",
+          "semanticVersion": "1.3.3",
+          "globalMessageStrings": {
+            "NotificationMessage": {
+              "text": "A notification was raised. Notification kind was: '{0}'."
+            },
```

Comment:
why so many changes u ask? its because for some reason we didnt have these samples included in the tests. I added a few test cases so that we can include these files as well (found 4 such cases in total in two different sets of tests). The changes in the file are due to rebaselining after a long time.